pyproject.toml

```diff
@@ -40,10 +40,12 @@ authors = [
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.7.4"
 typer = "*"
 aws-cdk-lib = "*"
 constructs = "*"
+aws-lambda-powertools = "^2.8.0"
+psycopg2-binary = "^2.9.5"
```

Comment:
If we're going to duplicate the dependencies from `lambda_functions/rds_init/requirements.txt` we probably want to set these to exact versions rather than [using caret](https://python-poetry.org/docs/dependency-specification/#caret-requirements).

Reply:
Ditto (as previous comment about managing Python packages for lambda). Will be a separate PR.